src/main/java/com/thebois/ColonyManagement.java

```diff
@@ -18,6 +20,12 @@
 import com.thebois.controllers.RoleController;
 import com.thebois.controllers.StructureController;
 import com.thebois.controllers.TerrainController;
+import com.thebois.models.Position;
+import com.thebois.models.beings.Colony;
+import com.thebois.models.beings.IBeing;
+import com.thebois.models.beings.Pawn;
+import com.thebois.models.beings.pathfinding.AstarPathFinder;
+import com.thebois.models.beings.pathfinding.IPathFinder;
```

Comment:
I just cleaned up our imports here and you do this 🙃 

On a serious note, this is sending me warning signals, since we refactored MVC to have less imports/coupling with ColonyManangment. I feel like a refactor where we move out colony from world should only add one more import to Colonymangemnet.

Reply:
On second thoughts, I don't mind Colonymanagement knowing about AstarPathFinder, but I feel like it should need to know about Pawns and IBeings.

Reply:
> I just cleaned up our imports here and you do this 🙃

Then we have plenty of room to import more stuff ! 😄 

But I think the question is, does creation and placing of pawns belong in colony? It is not completely wrong but it doesn't feel completely right either.

Reply:
This is an interesting question, it's probably best to discuss it in person tomorrow. I'll add a discussion item.